UnitsNet.Tests/BaseDimensionsTests.cs

```diff
@@ -0,0 +1,580 @@
+            Assert.True(result.Temperature == 0);
+            Assert.True(result.Amount == 0);
+            Assert.True(result.LuminousIntensity == 0);
+        }
+
+        [Fact]
+        public void CurrentDimensionsDivideCorrectlyWithOperatorOverloads()
+        {
+            var baseDimensions1 = new BaseDimensions(0, 0, 0, 5, 0, 0, 0);
+            var baseDimensions2 = new BaseDimensions(0, 0, 0, 4, 0, 0, 0);
+
+            var result = baseDimensions1 / baseDimensions2;
+
+            Assert.True(result.Length == 0);
+            Assert.True(result.Mass == 0);
+            Assert.True(result.Time == 0);
+            Assert.True(result.Current == 1);
+            Assert.True(result.Temperature == 0);
+            Assert.True(result.Amount == 0);
+            Assert.True(result.LuminousIntensity == 0);
+        }
+
+        [Fact]
+        public void TemperatureDimensionsDivideCorrectlyWithOperatorOverloads()
+        {
+            var baseDimensions1 = new BaseDimensions(0, 0, 0, 0, 4, 0, 0);
+            var baseDimensions2 = new BaseDimensions(0, 0, 0, 0, 3, 0, 0);
+
+            var result = baseDimensions1 / baseDimensions2;
+
+            Assert.True(result.Length == 0);
+            Assert.True(result.Mass == 0);
+            Assert.True(result.Time == 0);
+            Assert.True(result.Current == 0);
+            Assert.True(result.Temperature == 1);
+            Assert.True(result.Amount == 0);
+            Assert.True(result.LuminousIntensity == 0);
+        }
+
+        [Fact]
+        public void AmountDimensionsDivideCorrectlyWithOperatorOverloads()
+        {
+            var baseDimensions1 = new BaseDimensions(0, 0, 0, 0, 0, 3, 0);
+            var baseDimensions2 = new BaseDimensions(0, 0, 0, 0, 0, 2, 0);
+
+            var result = baseDimensions1 / baseDimensions2;
+
+            Assert.True(result.Length == 0);
+            Assert.True(result.Mass == 0);
+            Assert.True(result.Time == 0);
+            Assert.True(result.Current == 0);
+            Assert.True(result.Temperature == 0);
+            Assert.True(result.Amount == 1);
+            Assert.True(result.LuminousIntensity == 0);
+        }
+
+        [Fact]
+        public void LuminousIntensityDimensionsDivideCorrectlyWithOperatorOverloads()
+        {
+            var baseDimensions1 = new BaseDimensions(0, 0, 0, 0, 0, 0, 2);
+            var baseDimensions2 = new BaseDimensions(0, 0, 0, 0, 0, 0, 1);
+
+            var result = baseDimensions1 / baseDimensions2;
+
+            Assert.True(result.Length == 0);
+            Assert.True(result.Mass == 0);
+            Assert.True(result.Time == 0);
+            Assert.True(result.Current == 0);
+            Assert.True(result.Temperature == 0);
+            Assert.True(result.Amount == 0);
+            Assert.True(result.LuminousIntensity == 1);
+        }
+
+        [Fact]
+        public void CheckBaseDimensionDivisionWithSpeedEqualsDistanceDividedByTimeOnStaticPropertyWithOperatorOverloads()
```

Comment:
I like descriptive method names, but maaybe a bit overboard here? :-D
How about `LengthDividedByDurationReturnsSpeed` ? We are already in a `BaseDimensionsTests` class, so the context is implicit.

Again, we can address this in a separate PR. I'm about merge this in now.

Reply:
Excellent work on these tests though! 👏

Reply:
Perhaps a bit overly explicit :) Easy fix!